datafusion/core/src/execution/runtime_env.rs

```diff
@@ -121,6 +122,8 @@ pub struct RuntimeConfig {
     pub disk_manager: DiskManagerConfig,
     /// MemoryManager to limit access to memory
     pub memory_manager: MemoryManagerConfig,
+    /// ObjectStoreRegistry to get object store based on url
+    pub object_store_registry: ObjectStoreRegistry,
```

Comment:
This should possibly be `Arc<ObjectStoreRegistry>` for consistency

Reply:
Why? Other properties are not wrapped with Arc. And for one env, there should be only one runtime_env and its related properties.

Reply:
Two reasons:

* We use `Arc<ObjectStoreRegistry>` elsewhere and so using it here is more consistent
* It provides a hint that this is shared state, e.g. we use `Arc<DiskManager>` instead of just `DiskManager`, etc...

Reply:
I see. Agree with you that here better to use Arc